### ISSUE TITLE
No autocomplete for Model::create([])

### ISSUE THREAD

**Issue by Wulfheart** [simoebenhida/laravel-intellisense#23]
When using ``Model::create([/*attributes*/])`` there is no autocomplete for the attributes. Is this behaviour desired? How can I achieve autocomplete for the attributes there?

**Comment by simoebenhida** [comment]
Hi @Wulfheart 

```Model::create([/*attributes*/])``` Unfortunately is not yet supported!
Just added it to the list! Will close the issue when it is added!

**Comment by Wulfheart** [comment]
@simoebenhida I could try to implement it. Are there any major technical limitations which prevented you from implementing it?

**Comment by simoebenhida** [comment]
@Wulfheart Yeah it is a different feature! But it is doable! Just started working on it this weekend
